packages/appChrome/components/SidebarSection.tsx

```diff
@@ -27,14 +24,25 @@ class SidebarSection extends React.PureComponent<SidebarSectionProps, {}> {
   public render() {
     const { label, children } = this.props;
 
+    /* tslint:disable:no-string-literal */
+    const H3 = styled("h3")`
+      padding-left: ${props =>
+        spaceSizes[props.theme.sidebarHeaderPaddingHor] || spaceSizes["l"]};
+      padding-right: ${props =>
+        spaceSizes[props.theme.sidebarHeaderPaddingHor] || spaceSizes["l"]};
+      padding-bottom: ${props =>
+        spaceSizes[props.theme.sidebarHeaderPaddingVert] || spaceSizes["s"]};
+      padding-top: ${props =>
+        spaceSizes[props.theme.sidebarHeaderPaddingVert] || spaceSizes["s"]};
+    `;
+    /* tslint:enable:no-string-literal */
+
     return (
       <div>
         {label && (
-          <h3
+          <H3
```

Comment:
Just curious - why a capital "H"?
It doesn't matter to the browser, so I'm not requesting a change, just curious if there was a reason for changing this :)

Reply:
`H3` is my custom component - https://github.com/dcos-labs/ui-kit/pull/254/files/aa1bd95030f56ba0d105343df1d0d8b9da2c254a..97e3e540494c44b0a83eaceec0627653522285c1#diff-c449621d73b5c5204961ca0ba944428aR28 .